src/inputHandler.js

```diff
@@ -0,0 +1,35 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { ERROR_MESSAGES, INPUT_MESSAGES } from "./constants.js";
+
+export async function getValidatedCarNames() {
+  const CARS_INPUT = await MissionUtils.Console.readLineAsync(INPUT_MESSAGES.CAR_NAME_MESSAGE);
+  const CAR_NAMES = CARS_INPUT.split(",").map(carName => carName.trim());
+
+  if (CAR_NAMES.length === 0 || CAR_NAMES[0] === "") {
+    throw new Error(ERROR_MESSAGES.EMPTY_CAR_NAMES);
+  }
+
+  const UNIQUE_NAMES = new Set(CAR_NAMES);
+  if (UNIQUE_NAMES.size !== CAR_NAMES.length) {
+    throw new Error(ERROR_MESSAGES.DUPLICATE_CAR_NAMES);
+  }
+
+  CAR_NAMES.forEach((carName) => {
+    if (carName.length > 5) {
+      throw new Error(ERROR_MESSAGES.CAR_NAME_LENGTH);
+    }
+  });
```

Comment:
```suggestion
  CAR_NAMES.forEach((carName) => {
    if (carName.length > 5) {
      throw new Error(ERROR_MESSAGES.CAR_NAME_LENGTH);
    } else if (carName.length === ""){
      throw new Error(ERROR_MESSAGES.CAR_NAME_LENGTH);
    }
  });
```
이렇게 빈 문자열이 오는 경우도 캐치해야 할 것 같은데 아닌가요?

Reply:
앞쪽의 말씀해주신 split 함수의 최소 길이를 놓쳐 해당 부분이 빈 문자열이 오는 경우라고 생각해 `ERROR_MESSAGES.EMPTY_CAR_NAMES` 를 메세지로 하여 에러처리를 했던 것이었습니다. 

말씀주신대로 해당 부분에서 캐치해야할 것 같네요! ㅠㅠ